cryptography/src/bls12381/dkg/ops.rs

```diff
@@ -51,14 +51,31 @@ pub fn evaluate_all<V: Variant>(polynomial: &poly::Public<V>, n: u32) -> Vec<V::
     evals
 }
 
+/// A verified commitment.
+pub struct VerifiedCommitment<'a, V: Variant> {
```

Comment:
Like other parts of `dkg`, we try to make all flows "correct by construction" (and it shouldn't be possible to verify a share without a verified commitment).